appveyor.yml

```diff
@@ -19,12 +19,12 @@ cache:
 install:
     - ps: nuget install platyPS -Version 0.9.0 -source https://www.powershellgallery.com/api/v2 -outputDirectory "$Env:ProgramFiles\WindowsPowerShell\Modules\." -ExcludeVersion
     - ps: |
-        $requiredPesterVersion = '3.4.0'
+        $requiredPesterVersion = '4.1.1'
         $pester = Get-Module Pester -ListAvailable | Where-Object { $_.Version -eq $requiredPesterVersion }
         $pester
         if ($null -eq $pester) # WMF 4 build does not have pester
         {
-            cinst -y pester --version $requiredPesterVersion
+            nuget install Pester -Version 4.1.1 -source https://www.powershellgallery.com/api/v2 -outputDirectory "$Env:ProgramFiles\WindowsPowerShell\Modules\." -ExcludeVersion
```

Comment:
@JamesWTruher As I only noticed now in one of the new PRs, this does not seem to work for the `Visual Studio 2017` image because although it installs it OK, it still uses the old 3.4 version of Pester. I checked this by running `(Get-Command Invoke-Pester).Version`.